### PR TITLE
netgame: enable -loadgame

### PR DESCRIPTION
**Pull request by rfomin**

### CODE REVIEW

File: Source/g_game.c [hunk 1]
```diff
@@ -1711,7 +1711,6 @@ static void G_DoLoadGame(void)
 
   // killough 2/14/98: load compatibility mode
   compatibility = *save_p++;
-  demo_version = complevel;
```
Comment (fabiangreffrath) [inline Source/g_game.c:1714]: Oversight?

Comment (rfomin) [reply Source/g_game.c:1714]: Probably, I can't remember why I added this line.

**Comment by fabiangreffrath** [comment]
I somehow still don't get it. Didn't `-loadgame` work before already?

**Comment by rfomin** [comment]
> I somehow still don't get it. Didn't `-loadgame` work before already?

It worked for singleplayer. This PR enables it for netgame.

**Pull request merged**